### ISSUE TITLE
Content mixes with toolbar functions in editor

### ISSUE THREAD

**Issue by waridrox** [publiclab/plots2#9441]
### Please describe the problem (or idea)

**What happened just before the problem occurred? Or what problem could this idea solve?**

The writeup content of the post gets mixed up with the editor toolbar when scrolling or viewing content in general while making a post.

![FC586C05-ABF2-4D59-A9CB-1655D4D4A9DA_1_201_a](https://user-images.githubusercontent.com/58583793/113500746-266b2880-953e-11eb-9f6e-4a4d80378941.jpeg)

### Please show us where to look

https://publiclab.org/

### What's your PublicLab.org username?

waridrox

### Browser, version, and operating system

Chrome v89

**Comment by Priyaraj17** [comment]
Can you please tell me how to reproduce this error?

**Comment by waridrox** [comment]
Interestingly, the bug is not present on local env 🤔- 

https://user-images.githubusercontent.com/58583793/113585901-77684300-964a-11eb-966f-5ac0fdbff484.mp4

**Comment by TildaDares** [comment]
Can I work on this?

**Comment by waridrox** [comment]
@TildaDares sure 😄, go for it! 🚀

**Comment by TildaDares** [comment]
@waridox I was able to reproduce the error locally. I think a possible fix would be to make these buttons have an opaque background. 

## Before

<img width="922" alt="Screenshot 2021-04-08 at 15 34 31" src="https://user-images.githubusercontent.com/63427719/114045450-f0041500-987f-11eb-8fad-a6b9618f60db.png">

## After

<img width="922" alt="Screenshot 2021-04-08 at 15 38 04" src="https://user-images.githubusercontent.com/63427719/114046054-6dc82080-9880-11eb-9512-a3f42d89084d.png">

What do you think?

**Comment by waridrox** [comment]
Yes, @TildaDares you're thinking in the right direction! We don't require transparency on those buttons with the content.

**Comment by eightballocto** [comment]
This can be accomplished by adding this rule:
```css
.btn-light {
    background-color: white;
}
```

However, it should be noted that the Tags button is lacking the `.btn-light` class for whatever reason so more investigation is required.

**Comment by TildaDares** [comment]
@eightballocto The Toolbar already has a btn-light class but btn-outline-secondary is overriding it. Also I can't find this page in my local environment, can you please show me where it is?

**Comment by TildaDares** [comment]
Just to be sure these are the tags you're referring to.

![image](https://user-images.githubusercontent.com/63427719/114088656-273ceb00-98ad-11eb-9a50-459707fb7b7d.png)

**Comment by eightballocto** [comment]
> @eightballocto The Toolbar already has a btn-light class but btn-outline-secondary is overriding it. Also I can't find this page in my local environment, can you please show me where it is?

![image](https://user-images.githubusercontent.com/58581962/114089268-8bdf5200-987b-11eb-9baf-516e7766fa95.png)
using Inspect Element in my developer environment. It seems like these buttons are dynamically generated or from elsewhere. The "Tags" button I was referring to is in the toolbar, not the text field.

**Comment by waridrox** [comment]
Hi all, @eightballocto is right and there's a different repository https://github.com/publiclab/PublicLab.Editor dedicated to the editor functionality and I recommend to take a look at https://github.com/publiclab/PublicLab.Editor/issues/664 to anyone who is trying to come up with the fix. Thanks :)

**Comment by TildaDares** [comment]
@waridox How do I link this issue to my PR?

**Comment by Priyaraj17** [comment]
Hey, you just need to add the issue number in the PR template.
